config/routes.js

```diff
@@ -347,12 +347,7 @@ module.exports = [
   {
     method: 'POST',
     path: '/api/v3/user',
-    options: {
-      pre: [
-        UserPolicy.canCreate,
-      ],
-      handler: UserController.create,
-    },
+    handler: UserController.create,
```

Comment:
Your question sums up the purpose of the PR very well: the `canCreate` policy was doing a lot of unnecessary stuff that obscured the fact that anonymous API requests were allowed to create users.

So yes, the PR changes the permission logic:

- BEFORE: anonymous users, global managers, and admins could create new users (flawed policy because anyone can be anonymous by dropping the `Bearer` header)
- AFTER: now we can all clearly read from route config that there is no barrier to anyone creating a user account.

Reply:
Thanks for the clarification.